repos/rust-lang/compiler-team-prioritization.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "compiler-team-prioritization"
+description = "Home repository for the compiler team \"prioritization group\""
+bots = []
+
+[access.teams]
+wg-prioritization = "maintain"
```

Comment:
This seems to make sense, but I do wonder if `compiler` also needs access for any reason. cc @apiraino @wesleywiser

Reply:
I can't think of any reason `compiler` or `compiler-contributors` would need it off hand 🙂